sos/report/plugins/openstack_nova.py

```diff
@@ -71,17 +71,16 @@ def setup(self):
                                     "intended to connect to the OpenStack "
                                     "environment.")
             else:
-                self.add_cmd_output("nova service-list")
+                self.add_cmd_output("openstack compute service list")
                 self.add_cmd_output("openstack flavor list --long")
-                self.add_cmd_output("nova network-list")
-                self.add_cmd_output("nova list --all-tenants")
-                self.add_cmd_output("nova agent-list")
```

Comment:
The `nova` commands are deprecated? Shall we keep them for some time for legacy reasons, or are they gone for a while?

Reply:
The `nova` commands are not deprecated, and still exist, some sub commands no longer exist, and more of the features are being added to the `openstack` command. So makes sense to move as many over. At least that was my thought process here

Reply:
I agree with @arif-ali . I guess that the only command deprecated there is `nova network-list`

Reply:
Ack, if openstack is moving these commands over time and the idea is that at _some_ point the `nova` commands will be removed, it's good to get ahead of the game.